sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -1240,5 +1608,127 @@ await using (var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsum
                 }
             }
         }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubConsumer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        [Ignore("Identifiers are not supported in Track Two yet")]
```

Comment:
I'm not sure what isn't supported;  you're setting them as part of the options.   Was there something specific that you found missing?  Just setting the property for informational use?   We can add that easily enough, if so.

Reply:
I can set it, but I couldn't read it back. In Track One, this was possible by calling `receiver.Identifier`. Are we doing it another way in Track Two?

Reply:
Given that I know the identifiers' names, I can update the test so it doesn't need to call `consumer.Identifier`. But I'm still curious about the question above.

Reply:
There wasn't a property exposing;  I just added that to the consumer.